src/utils/utils.rs

```diff
@@ -615,6 +634,14 @@ where
                     notify_handler(Notification::RenameInUse(&src, &dest).into());
                     OperationResult::Retry(e)
                 }
+                io::ErrorKind::Other
+                    if cfg!(target_os = "linux") && Some(18i32) == e.raw_os_error() =>
```

Comment:
written like this, this code path is compiled into every binary, linux or not; I think a #[cfg(linux)] block would be better. 

Rather than having the magic number 18 here, please us https://docs.rs/libc/0.2.72/libc/constant.EXDEV.html